examples/great_expectations_examples/ge_expect_column_value_confidence_for_data_label.ipynb

```diff
@@ -0,0 +1,138 @@
+{
```

Comment:
let's build more of a story line around this example... give the end user something to connect with / see the use case for this functionality in high definition using a storyline